src/main/resources/anniv5Resources/localization/eng/dimensiongatepack3/Cardstrings.json

```
@@ -8,28 +8,28 @@
   },
   "${ModID}:Inferno": {
     "NAME": "Inferno",
-    "DESCRIPTION": "Deal !D! damage and apply 10 ${ModID}:Ignite to ALL enemies. NL Gain !M! ${ModID}:Ignite."
+    "DESCRIPTION": "Deal !D! damage and apply !M! ${ModID}:Ignite to ALL enemies. NL Add 2 *Burns to your hand."
   },
   "${ModID}:Minniegun": {
     "NAME": "Minniegun",
     "DESCRIPTION": "Deal !D! damage !M! times. NL Put a *Jam in your discard pile."
   },
   "${ModID}:Ouroboros": {
     "NAME": "Ouroboros",
-    "DESCRIPTION": "At the end of each turn, if you fully blocked any source of damage, gain 1 Strength.",
-    "UPGRADE_DESCRIPTION": "Innate. NL At the end of each turn, if you fully blocked any source of damage, gain 1 Strength."
+    "DESCRIPTION": "The first time you fully block\u200b any source of damage each turn, gain 1 Strength.",
```

Going to make a set of edits to fix the block keyword thing in a slightly different way. The original way I suggested was behaving inconsistently for reason I don't fully understand.
```suggestion
    "DESCRIPTION": "The first time you fully block\u00a0any source of damage each turn, gain 1 Strength.",
```